### ISSUE TITLE
Related media heading sizes different in audio/image pages

### ISSUE THREAD

**Issue by obulat** [WordPress/openverse#3863]
## Problem

The `Related images` uses a Heading 5 for screens above `md` and the `Related audio` always uses a Heading 6. These both should probably be Heading 5 above `md`, @fcoveram ?

![image](https://github.com/WordPress/openverse/assets/6351754/2769fcdf-f7a3-4814-b38b-8a012e8c81ae)

## Additional context

First reported by @zackkrida : https://github.com/WordPress/openverse/pull/3831#discussion_r1509533709

This issue is blocked because it needs input from @fcoveram

**Comment by fcoveram** [comment]
Yes, you are right. Heading 5 for screens above `md` in both pages.

**Comment by Mang0codes** [comment]
Hi! I would like to work on this.

**Comment by tejaswarathe** [comment]
Hi, I would like to work on this issue
